lib/StaticNarrative/exporter/data_exporter.py

```diff
@@ -75,10 +73,10 @@ def export_narrative_data(
         "types": type_info,
     }
 
-    output_path = os.path.join(output_dir, "data.json")
+    output_path = Path(output_dir) / "data.json"
```

Comment:
this is mostly because the pytest `tmpdir` fixture is a `Path` object - may as well use the `Path` module instead of `os.path`

Reply:
I've started using Pathlib much more often in other projects and find it much more intuitive to use.